### PR TITLE
Elasticstack

### PR DESCRIPTION
**Pull request by ninjeanne**

### CODE REVIEW

**Comment by mor4thii** [comment]
Could you please rebase onto main? Sorry for misusing main yesterday.
Also, if possible, please add the issue at the end of each commit message: <emoji> <Description> (#<issue number>)

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  es01:
+    profiles: ["all", "elastic-only"]
+    depends_on:
+      setup:
+        condition: service_healthy
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTICSTACK_VERSION}
+    networks:
+      - elk
+    ports:
+      - 9200:9200
```
Comment (mor4thii) [inline docker/docker-compose.yml:56]: Please always specify port mappings as "xx:yy", not xx:yy (https://docs.docker.com/compose/compose-file/compose-file-v3/#ports)

File: docker/humlib.profiles [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export HUMLIB_BE_HOME=<YOUR PATH>
```
Comment (mor4thii) [inline docker/humlib.profiles:1]: Can we offer these as IntelliJ run configurations?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,52 @@
...
+## SDKman
+
+For automatically using Java 17 when entering this repository, simply set `sdkman_auto_env=true`
+in `~/.sdkman/etc/config`
+This makes sdkman switch to the java version defined in `.sdkmanrc` automatically when you `cd` into this repo.
+
+## elasticstack
+version: [8.0.0](https://www.elastic.co/guide/en/elastic-stack-get-started/8.0/get-started-stack-docker.html#run-docker-secure "documentation")
+
+### prerequisites
```
Comment (mor4thii) [inline README.md:30]: Capitalise all headlines, please

File: README.md [hunk 4]
```diff
@@ -1,15 +1,52 @@
...
+- docker desktop
+- at least 4GB available memory
+
+### setup
+- open a terminal
+- navigate to the directory `humlib-be/docker`
```
Comment (mor4thii) [inline README.md:36]: I think that either ./docker, /docker, or just docker is sufficient.

File: README.md [hunk 5]
```diff
@@ -1,15 +1,52 @@
...
+- copy `humlib.profiles` to `humlib.profiles.overrides` and set the absolute path to your "humlib-be"-project
```
Comment (mor4thii) [inline README.md:37]: "to your humlib-be project" or just "to this repository"

File: docker/.env.local [hunk 6]
```diff
@@ -0,0 +1,6 @@
+ELASTICSTACK_VERSION="8.0.0"
+USER_ELASTIC_PASSWORD="huml1b_el4st1c"
+USER_KIBANA_SYSTEM_PASSWORD="huml1b_k1b4n4"
+
+# Increase or decrease based on the available host memory (in bytes)
+ELASTICSTACK_MEM_LIMIT="1073741824"
```
Comment (mor4thii) [inline docker/.env.local:6]: Can you mention this in README, please? That you can increase this locally if your PC can handle it?

File: README.md [hunk 7]
```diff
@@ -1,15 +1,52 @@
...
+This makes sdkman switch to the java version defined in `.sdkmanrc` automatically when you `cd` into this repo.
+
+## elasticstack
+version: [8.0.0](https://www.elastic.co/guide/en/elastic-stack-get-started/8.0/get-started-stack-docker.html#run-docker-secure "documentation")
+
+### prerequisites
+- docker desktop
+- at least 4GB available memory
+
+### setup
```
Comment (mor4thii) [inline README.md:34]: I had some troubles getting Chrome to open elasticsearch on localhost:9200 due to certificate issues. I was able to fix this by manually adding the generated certificate from docker/volumes to Keychain and trusting it.

**Review by mor4thii — CHANGES_REQUESTED** [review]
First review done.

File: docker/humlib.profiles (hunk 2 above)
Comment (ninjeanne) [reply docker/humlib.profiles:1]: how :o

Comment (mor4thii) [reply docker/humlib.profiles:1]: Let‘s just write an issue and do it later. It would only be convenience.

**Comment by mor4thii** [comment]
We will merge keycloak and elastic in one.

**Pull request closed without merging**